lib/model/name.dart

```diff
@@ -0,0 +1,19 @@
+class Name {
+  Name({
+    required this.title,
+    required this.first,
+    required this.last,
+  });
+
+  factory Name.fromJson(Map<String, dynamic> json) {
+    return Name(
+      title: json['title'],
+      first: json['first'],
+      last: json['last'],
+    );
+  }
+
+  late String title;
+  late String first;
+  late String last;
```

Comment:
why did you defined this values as late?

Reply:
because it throws an error if I don't define it as late

Reply:
now i change it